### ISSUE TITLE
when trying to compile the Blink example, I get error message: "unable to rename 'core\core.a'; reason: File exists". I have no idea how to resolve this. Can anyone help?

### ISSUE THREAD

**Issue by jwente49** [FastLED/FastLED#636]

**Comment by focalintent** [comment]
I need a bit more context - what hardware are you trying to compile for, and what version of Arduino are you using?  core/core.a sounds like something that is way outside of the realm of what FastLED references.

**Comment by jwente49** [comment]
I found that usually I can just compile again and it doesn't give this error. Found other users had same experience.  Sounds to me like a bug in the Arduino app. Now happens on other scripts I wrote years ago. It never happened back then, but that was an older version of the app.On Aug 8, 2018 4:33 PM, Daniel Garcia <notifications@github.com> wrote:I need a bit more context - what hardware are you trying to compile for, and what version of Arduino are you using?  core/core.a sounds like something that is way outside of the realm of what FastLED references.
